apps/authentication-portal/src/main/webapp/org_name.jsp

```diff
@@ -0,0 +1,155 @@
+<%@ include file="includes/localize.jsp" %>
+<%@ include file="includes/init-url.jsp" %>
+
+<%
+
+    String idp = request.getParameter("idp");
+    String authenticator = request.getParameter("authenticator");
+    String sessionDataKey = request.getParameter(Constants.SESSION_DATA_KEY);
+    
+    String errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "error.retry");
+    String authenticationFailed = "false";
+    
+    if (Boolean.parseBoolean(request.getParameter(Constants.AUTH_FAILURE))) {
+        authenticationFailed = "true";
+        
+        if (request.getParameter(Constants.AUTH_FAILURE_MSG) != null) {
+            errorMessage = request.getParameter(Constants.AUTH_FAILURE_MSG);
+            
+            if (errorMessage.equalsIgnoreCase("authentication.fail.message")) {
+                errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "error.retry");
+            }
+        }
+    }
+%>
+
+<html>
+    <head>
+        <!-- header -->
+        <%
+            File headerFile = new File(getServletContext().getRealPath("extensions/header.jsp"));
+            if (headerFile.exists()) {
+        %>
+                <jsp:include page="extensions/header.jsp"/>
+        <%
+            } else {
+        %>
+                <jsp:include page="includes/header.jsp"/>
+        <%
+            }
+        %>
+
+        <!--[if lt IE 9]>
+        <script src="js/html5shiv.min.js"></script>
+        <script src="js/respond.min.js"></script>
+        <![endif]-->
+    </head>
+
+    <body class="login-portal layout authentication-portal-layout">
+        <main class="center-segment">
+            <div class="ui container medium center aligned middle aligned">
+                <!-- product-title -->
+                <%
+                    File productTitleFile = new File(getServletContext().getRealPath("extensions/product-title.jsp"));
+                    if (productTitleFile.exists()) {
+                %>
+                        <jsp:include page="extensions/product-title.jsp"/>
+                <%
+                    } else {
+                %>
+                        <jsp:include page="includes/product-title.jsp"/>
+                <%
+                    }
+                %>
+
+                <div class="ui segment">
+                    <!-- page content -->
+                    <h2>Sign In with Enterprise Login</h2>
+                    <div class="ui divider hidden"></div>
+
+                    <%
+                        if ("true".equals(authenticationFailed)) {
+                    %>
+                            <div class="ui negative message" id="failed-msg"><%=Encode.forHtmlContent(errorMessage)%></div>
+                            <div class="ui divider hidden"></div>
+                    <%
+                        }
+                    %>
+
+                    <div id="alertDiv"></div>
+
+                    <div class="segment-form">
```

Comment:
Can we avoid this segment? I think the elevated view does not look alright to me IMO.

Reply:
fixed.